### PR TITLE
[12.0] [FIX] The Default Operating Unit of the user is not assigned to it.

### PR DESCRIPTION
**Pull request by nikul-serpentcs**
Issue: #333

### CODE REVIEW

**Review by AaronHForgeFlow — APPROVED** [review]
LGTM @Dangar-Serpentcs could you check it is solved on your side?

File: product_operating_unit/models/product_template.py [hunk 1]
```diff
@@ -39,8 +39,9 @@ def _check_operating_unit(self):
     @api.onchange('categ_id')
     def onchange_operating_unit_ids(self):
         for record in self:
-            record.operating_unit_ids = \
-                [(6, 0, record.categ_id.operating_unit_ids.ids)]
+            if record.categ_id and record.categ_id.operating_unit_ids:
```
Comment (Swapnesh-SerpentCS) [inline product_operating_unit/models/product_template.py]:
```suggestion
            if record.categ_id.operating_unit_ids:
```

Comment (nikul-serpentcs) [reply product_operating_unit/models/product_template.py]: @Swapnesh-SerpentCS Done, Please re-review

**Comment by Dangar-Serpentcs** [comment]
Getting another trace-back when going to `Save` the Product.

Screenshot from 2020-10-20 14-42-01

**Comment by nikul-serpentcs** [comment]
@Dangar-Serpentcs I check in my local and Works Fine.
Product_OU_Peek 2020-10-20 14-51

**Comment by Dangar-Serpentcs** [comment]
LGTM +1

**Comment by AaronHForgeFlow** [comment]
@Swapnesh-SerpentCS @Dangar-Serpentcs Could you do a former review from the reviews button? Thank you!

**Review by Swapnesh-SerpentCS — APPROVED** [review]

**Comment by nikul-serpentcs** [comment]
@AaronHForgeFlow Ready to merge

**Pull request merged**